engines/director/lingo/lingo-the.cpp

```diff
@@ -758,7 +758,8 @@ Datum Lingo::getTheEntity(int entity, Datum &id, int field) {
 		d.u.i = _vm->getVersion();
 		break;
 	case kTheSearchCurrentFolder:
-		getTheEntitySTUB(kTheSearchCurrentFolder);
+		//We always allow searching in current folder
```

Comment:
Please issue a warning here, something like `"BUILDBOT: SearchCurrentFolder is queried"`.

And as discussed add a similar warning to the setting of the property, so we can catch when it is used.